### PR TITLE
Blog post about cost of Perma

### PR DESCRIPTION
**Pull request by kilbergr**

### CODE REVIEW

File: app/_posts/2024-02-08-the-cost-of-a-digital-archive.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+How do you use those numbers to calculate the cost of storing something forever? One way to get at that answer is by performing a [capitalized cost analysis](https://global.oup.com/us/companion.websites/9780190296902/sr/interactive/worth/sec4/). A capitalized cost analysis determines the current value of something that will exist on a long timescale. The formula for capitalized cost is `up-front cost + maintenance cost \* (1/interest rate)`. In our case, we don’t have an interest rate, but we can make a fairly similar calculation, taking the nominal annual payment (p), the rate of inflation (r), and time (t). For each year, we’ll take `p \* (1 - r)^t`, then we add them up to infinity. In this chart, infinity is represented by 100 years, and we’ve graphed the cost of 1 link, 10 links, and 100 links over time.
```
Comment (bensteinberg) [inline app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: I'm not sure you want the backslashes in front of the multiplication asterisks here.

**Review by bensteinberg — COMMENTED** [review]
This is great! See the one comment about line 16.

File: app/_posts/2024-02-08-the-cost-of-a-digital-archive.md (hunk 1 above)
Comment (kilbergr) [reply app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: When I was using the LIL blog generator, the * turned into italics, so I thought I needed to escape them. I'll take them out.

Comment (bensteinberg) [reply app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: I'll just check it locally, too, I should have done that already.

Comment (kilbergr) [reply app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: I just removed them--I think I don't need them because I put those lines in `code format`

Comment (bensteinberg) [reply app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: Great. I see you added `sharing-image: images/PermaCost_Figure.png` to the frontmatter -- I think you then need to put `PermaCost_Figure.png` in `app/assets/images/`.

Comment (kilbergr) [reply app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: Ahh. OK I will do that. One sec.

Comment (bensteinberg) [reply app/_posts/2024-02-08-the-cost-of-a-digital-archive.md]: > When I was using the LIL blog generator, the * turned into italics, so I thought I needed to escape them. I'll take them out.

How did you get the code syntax to work in the blog generator? With single backticks? I just tried it and couldn't get it to show italics inside the code section.

**Pull request merged**